course/frontend/arrays/scripts/main.js

```diff
@@ -93,4 +93,38 @@ function recursiveArrayClone(source) {
     return result;
 }
 
-console.log('1) recursiveArrayClone([1, 2, 3, [1, 2, 3]])', res = recursiveArrayClone(arr), 'and should be a new array', res != arr, 'and sub-array should be a new array', arr[3] != res[3]);
\ No newline at end of file
+console.log('1) recursiveArrayClone([1, 2, 3, [1, 2, 3]])', res = recursiveArrayClone(arr), 'and should be a new array', res != arr, 'and sub-array should be a new array', arr[3] != res[3]);
+
+//26. Write a JavaScript function to generate an array between two integers of 1 step length.
+
+function rangeBetwee(start, end) {
```

Comment:
Robert,
Te envío una forma mas reducida de realizar el ejercicio:

`function rangeBetwee(init, fin) {
    var array = [];
    for (var i = init; i <= fin; i++) {
        array.push(i);
    }
    return array;
}`

Reply:
Vale, Jaime cambiado gracias!